index.js

```diff
@@ -7,8 +7,7 @@ const fs = require("fs"),
     session = require("express-session"),
     cors = require("cors"),
     passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+    errorhandler = require("errorhandler");
```

Comment:
Please, we can stick to ES6 syntax of `import` instead of `require`.

Reply:
There is a chore on the PT board that requires Samuel and Olumide to change the codebase to  ES6 syntax, that's the reason I didn't convert it.